bundles/org.openhab.binding.modbus.sunspec/src/main/resources/ESH-INF/thing/inverter-types.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="inverter-single-phase">
```

Comment:
Do you plan to support three-phase inverter ? I can provider modbus/tcp access to mine if needed.

Best regards,

a.

Reply:
> Do you plan to support three-phase inverter ? I can provider modbus/tcp access to mine if needed.

Thanks, yes, I already had implemented it in the full version, but then I have split it up into smaller PR-s for easier review. This PR contains only the most basic functionality and focuses on the skeleton of the bundle.

2/3 phase inverters are then almost just repetitions of the same components. 

After hopefully this bundle gets accepted I'll submit the autodiscovery feature, after those I can start add more device specific functions.